src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/producer/AvroSerializer.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.companieshouse.chipsrestinterfacesconsumer.producer;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericDatumWriter;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.io.BinaryEncoder;
+import org.apache.avro.io.EncoderFactory;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.model.ChipsKafkaMessage;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+@Component
+public class AvroSerializer {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
this line can be removed, objectMapper is never used

Reply:
Removed